spec/support/jasmine.json

```diff
@@ -0,0 +1,11 @@
+{
+  "spec_dir": "spec",
+  "spec_files": [
+    "**/*[sS]pec.js"
+  ],
+  "helpers": [
+    "helpers/**/*.js"
+  ],
+  "stopSpecOnExpectationFailure": false,
+  "random": false
```

Comment:
Is there a particular reason we're using `random: false`? I think in general randomized test order is a good thing.